src/scripts/specialEvents/SpecialEvents.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference path="../../declarations/GameHelper.d.ts" />
 /// <reference path="../../declarations/DataStore/common/Feature.d.ts" />
+/// <reference path="../../declarations/specialEvents/specialEvent.d.ts" />
```

Comment:
This file doesn't exist, the actual declaration file has a capital S on specialEvent.d.ts
It doesn't seem needed for the file to get type information, deleting the line doesn't cause any issues, but maybe it makes it slightly more efficient? I have only been adding these references when it complains about not knowing what something is.

Reply:
I found the reason i did this. I added the reference path because when you save the file and click somewhere with the reference removed it display and error for a little bit (the time depended on how much stuff i have open on my laptop which is usually a lot) and complains that you dont have it. it fixes itself but i just noticed it now. Also, the typo was something i noticed but didnt push here. I removed it as it seems fine without it? I wasnt getting anything in the console or eslint-fix. heres a screen of the split second errors just because. 
![image](https://user-images.githubusercontent.com/49108377/103163209-faf2d800-47af-11eb-880c-281ae6aa76d6.png)